lib/std/Thread/Futex.zig

```diff
@@ -264,7 +264,7 @@ const LinuxImpl = struct {
 
         const rc = linux.futex_wait(
             @as(*const i32, @ptrCast(&ptr.raw)),
-            linux.FUTEX.PRIVATE_FLAG | linux.FUTEX.WAIT,
```

Comment:
What is the performance cost of this?
I don't think this should be changed here, this is a cross-platform Futex implementation (which I would guess does not include multi-process sharing.)